### ISSUE TITLE
I got a 100% accuracy in ucf101 test_list2 and test_list3 by using ucf101_rgb weights

### ISSUE THREAD

**Issue by DUANJINHAO** [shuangshuangguo/caffe2pytorch-tsn#17]
Hi, thanks for your great job.
I got a comparable result on ucf101-split1 after loading ucf101_rgb.pth weights. Strangely, I got 100% accuracy on split2 and split3 respectively. All the results were computed by using test_models.py provided from tsn-pytorch repository, and I'm sure it works.
Finally, I got 95.33% accuracy averaging on these three splits which is totally different from the paper(Only RGB). 
Could you please give me the results that you test on split2 and split3?
Thanks again.

**Comment by shuangshuangguo** [comment]
The model given in this repo is trained on split1, so you need to test it on split1 too, not split2 and split3.
I do not support models on split2 and split3.

**Comment by DUANJINHAO** [comment]
All right, thaks :)

**Comment by DUANJINHAO** [comment]
Hi @shuangshuangguo 
For UCF101 dataset, there is an official classInd.txt file. 
But I cannot find this file in HMDB51. Could you please tell me where to get that? 
Thanks

**Comment by DUANJINHAO** [comment]
Looks like in an alphabetic order.

**Comment by shuangshuangguo** [comment]
Yes, you are right.
![image](https://user-images.githubusercontent.com/31126075/57310712-d64e7600-711c-11e9-800f-02e1b8f2fc6f.png)
